packages/expo-updates/android/src/main/java/expo/modules/updates/loader/FileDownloader.kt

```diff
@@ -485,7 +506,21 @@ open class FileDownloader(context: Context) {
             header(key, value)
           }
         }
+        .apply {
+          configuration.codeSigningConfiguration?.let {
+            header("expo-expects-signature", Crypto.createAcceptSignatureHeader(it))
```

Comment:
It actually looks like we might be able to use `expo-accept-signature` since the one used for classic Expo Go manifest signing is `exponent-accept-signature`.

Reply:
Actually wait nevermind, it uses `exponent-accept-signature` for classic manifests and `expo-accept-signature` for new manifests.